src/en/mayotune/src/eu/kanade/tachiyomi/extension/en/mayotune/MayoTune.kt

```diff
@@ -0,0 +1,128 @@
+package eu.kanade.tachiyomi.extension.en.mayotune
+
+import eu.kanade.tachiyomi.network.GET
+import eu.kanade.tachiyomi.source.model.FilterList
+import eu.kanade.tachiyomi.source.model.MangasPage
+import eu.kanade.tachiyomi.source.model.Page
+import eu.kanade.tachiyomi.source.model.SChapter
+import eu.kanade.tachiyomi.source.model.SManga
+import eu.kanade.tachiyomi.source.online.HttpSource
+import eu.kanade.tachiyomi.util.asJsoup
+import keiyoushi.utils.parseAs
+import okhttp3.Request
+import okhttp3.Response
+import rx.Observable
+
+class MayoTune() : HttpSource() {
+    override val name: String = "MayoTune"
+    override val baseUrl: String = "https://mayotune.xyz"
+    override val lang: String = "en"
+    override val versionId: Int = 1
+
+    private val source = SManga.create().apply {
+        title = "Mayonaka Heart Tune"
+        url = "/"
+        thumbnail_url = "$baseUrl/img/cover.jpg"
+        author = "Masakuni Igarashi"
+    }
+
+    // Popular
+    override fun fetchPopularManga(page: Int): Observable<MangasPage> {
+        return Observable.just(MangasPage(listOf(source), false))
+    }
+
+    override fun popularMangaRequest(page: Int): Request = throw UnsupportedOperationException()
+    override fun popularMangaParse(response: Response): MangasPage =
+        throw UnsupportedOperationException()
+
+    // Latest
+    override val supportsLatest: Boolean = true
+    override fun fetchLatestUpdates(page: Int): Observable<MangasPage> {
+        return Observable.just(MangasPage(listOf(source), false))
+    }
+
+    override fun latestUpdatesRequest(page: Int): Request = throw UnsupportedOperationException()
+    override fun latestUpdatesParse(response: Response): MangasPage =
+        throw UnsupportedOperationException()
+
+    // Search
+    override fun fetchSearchManga(
+        page: Int,
+        query: String,
+        filters: FilterList,
+    ): Observable<MangasPage> {
+        val mangas = mutableListOf<SManga>()
+
+        if (source.title.lowercase().contains(query.lowercase()) ||
+            source.author?.lowercase()?.contains(query.lowercase()) == true
+        ) {
+            mangas.add(source)
+        }
+
+        return Observable.just(MangasPage(mangas, false))
+    }
+
+    override fun searchMangaRequest(page: Int, query: String, filters: FilterList) =
+        throw UnsupportedOperationException()
+
+    override fun searchMangaParse(response: Response): MangasPage =
+        throw UnsupportedOperationException()
+
+    // Get Override
+    override fun mangaDetailsRequest(manga: SManga): Request {
+        return GET(manga.url, headers)
+    }
+
+    override fun chapterListRequest(manga: SManga): Request {
+        return GET(manga.url + "/api/chapters", headers)
+    }
```

Comment:
Don't you also need to change this so it handles the relative URL?
```suggestion
    override fun chapterListRequest(manga: SManga): Request {
        return GET("$baseUrl${manga.url}api/chapters", headers)
    }
```

Reply:
Apparently the URL in the overwritten method gets concatenated with `baseUrl` as I've mentioned before.
I've tested it myself before committing and it's fine.

Reply:
> Removing the manga details override breaks the extension for the same reason as it tries to resolve into absolute URLs and concats them anyway(?)

This is what I was talking about:

![Screenshot_20250609_030933_Mihon.jpg](https://github.com/user-attachments/assets/5574f312-6f4e-4154-9fc1-54d89411f582)

Reply:
Right, but that might be a leftover from the entry being in the database already. From a fresh state I would imagine it acts differently.

Reply:
You are welcome to test it yourself.
The cache may be there at launch, but errors will show up as soon as you refresh the entry if you decide to remove the override methods. After all, endpoints are part of the extension code, not cache, yes?

Reply:
I tested the changes on a fresh AVD and indeed it did not concatentate `baseURL` by default. I don't get *why* that behaviour is cached and changes were ignored but you were right, my apologies.